### PR TITLE
RPA-123: Add Daniel bio

### PR DESCRIPTION
**Pull request by danielsojoes**

### CODE REVIEW

**Review by javiervieramolina — CHANGES_REQUESTED** [review]
Please, define your favorite cereal

File: SkillsBoosterWorkshops/GitTraining/participants/danielsojo.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<participant>
+  <name>Daniel Sojo</name>
+  <favoriteCereal>TBD</favoriteCereal>
```
Comment (alvaro-navio) [inline SkillsBoosterWorkshops/GitTraining/participants/danielsojo.xml]: You should define one

File: SkillsBoosterWorkshops/GitTraining/participants/danielsojo.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<participant>
+  <name>Daniel Sojo</name>
```
Comment (eduardo-fuentes) [inline SkillsBoosterWorkshops/GitTraining/participants/danielsojo.xml:2]: Your name has a typ
```suggestion
  <name>Daniel SoJo</name>
```

**Review by eduardo-fuentes — CHANGES_REQUESTED** [review]

File: SkillsBoosterWorkshops/GitTraining/participants/danielsojo.xml (hunk 1 above)
Comment (eduardo-fuentes) [inline SkillsBoosterWorkshops/GitTraining/participants/danielsojo.xml]: You are missing your favorite cereal. 
```suggestion
  <favoriteCereal>Corn Flakes</favoriteCereal>
```

**Review by eduardo-fuentes — CHANGES_REQUESTED** [review]

**Review by alvaro-navio — CHANGES_REQUESTED** [review]

File: SkillsBoosterWorkshops/GitTraining/participants/danielsojo.xml (hunk 1 above)
Comment (danielsojoes) [reply SkillsBoosterWorkshops/GitTraining/participants/danielsojo.xml]: Already updated.

**Review by alvaro-navio — COMMENTED** [review]
Perfect

**Pull request closed without merging**